GNUmakefile

```diff
@@ -11,7 +11,7 @@ install:
 		(test -f terraform-provider-artifactory || go build -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}'") && \
 		mv terraform-provider-artifactory terraform.d/plugins/registry.terraform.io/jfrog/artifactory/${NEXT_VERSION}/darwin_amd64 && \
 		rm -f .terraform.lock.hcl && \
-		sed -i 's/version = ".*"/version = "${NEXT_VERSION}"/' sample.tf && \
+		sed -i.bak 's/version = ".*"/version = "${NEXT_VERSION}"/' sample.tf && rm sample.tf.bak && \
```

Comment:
I tried doing a make yesterday and I was getting parsing errors. Are you trying this on mac?

Reply:
Yes. I was trying this on mac. Previous one failed for me in mac. i checked this one on mac and linux/ubuntu.
Are you also on mac?

Reply:
```bash
christianb@christianb-mac terraform-provider-artifactory % make install
mkdir -p terraform.d/plugins/registry.terraform.io/jfrog/artifactory/2.7.1/darwin_amd64 && \
                (test -f terraform-provider-artifactory || go build -ldflags="-X 'github.com/jfrog/terraform-provider-artifactory/pkg/artifactory.Version=2.7.1'") && \
                mv terraform-provider-artifactory terraform.d/plugins/registry.terraform.io/jfrog/artifactory/2.7.1/darwin_amd64 && \
                rm -f .terraform.lock.hcl && \
                sed -i 's/version = ".*"/version = "2.7.1"/' sample.tf && \
                terraform init
sed: 1: "sample.tf": unterminated substitute pattern
make: *** [install] Error 1
```

Reply:
Using a backup file is one alternative 😄

Reply:
Let's just have it use docker. Then we know we can get consistency